index.html

```diff
@@ -4,24 +4,43 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Its a book collection website where you can track of the you road already">
     <script defer src="js/apps.js"></script>
     <title>Awesome Books</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lobster&display=swap"
+      rel="stylesheet"
+    />
 
     <link rel="stylesheet" href="style/style.css" />
   </head>
   <body>
+    <header>
+      <div class="logo"><a href="/">Awesome Books</a></div>
```

Comment:
Hello @Lucas-Masaba, Can you please highlight it for me? it's a bit tricky. don't get what you mean. as long as We wanted it to be like when the user clicks on the title to display everything when clink on the list display online list. Thanks for making it clear to us Please

Reply:
Hello @Bateyjosue , when I click the 'Awesome Books' text on the top left of the page, the views from the list, add new and contact are displayed on the same page and yet they are supposed to be in there respective pages. The link on the 'Awesome Books' pages should be able to link to the list which is the home page or have no link at all. I do not know if that makes sense but in case it does not feel free to leave a reply.